aiida_raspa/calculations/__init__.py

```diff
@@ -174,6 +174,7 @@ def _handle_retrieved_parent_folder(self, inp, folder):
             if system["type"] == "Box":
                 system_or_box = "Box"
                 (n_x, n_y, n_z) = (1, 1, 1)
+                system['ExternalPressure'] = 0
```

Comment:
```suggestion
            if 'ExternalPressure' not in system:
                system['ExternalPressure'] = 0
```
This would be better

Reply:
done.